Makefile

```diff
@@ -15,9 +18,21 @@ build:
 	@docker build -t $(IMAGE_NAME):latest -f dockerfiles/Dockerfile .
 	@docker tag $(IMAGE_NAME):latest $(IMAGE_NAME):$(IMAGE_TAG)
 
+build-dbs:
+	@docker build -t $(POSTGRES_IMAGE_NAME):latest -f dockerfiles/Dockerfile.postgres .
+	@docker tag $(POSTGRES_IMAGE_NAME):latest $(POSTGRES_IMAGE_NAME):$(IMAGE_TAG)
+	@docker build -t $(MDB_IMAGE_NAME):latest -f dockerfiles/Dockerfile.mariadb .
+	@docker tag $(MDB_IMAGE_NAME):latest $(MDB_IMAGE_NAME):$(IMAGE_TAG)
+
 push:
 	@docker --config=$(DOCKER_CONF) push $(IMAGE_NAME):latest
-	@docker --config=$(DOCKER_CONF) push $(IMAGE_NAME):$(IMAGE_TAG)
+	@docker --config=$(DOCKER_CONF) push $(IMAGE_NAME):latest
```

Comment:
Is this a typo? We don'T want to change how the qontract-reconcile (IMAGE_NAME) is pushed. This would effectively remove the hash based tag (IMAGE_TAG) and push :latest twice?

Reply:
I fixed it in the latest commit